Packs/Pulsedive/Integrations/Pulsedive/Pulsedive.py

```diff
@@ -168,46 +192,68 @@ def domain_reputation_command(client: Client, args: Dict[str, Any], api_key) ->
-        reputation = domain_data['risk']
-        score = convert_to_xsoar_severity(reputation)
-
-        if 'creation_date' in domain_data:
-            domain_data['creation_date'] = parse_domain_date(domain_data['creation_date'])
-        if 'expiration_date' in domain_data:
-            domain_data['expiration_date'] = parse_domain_date(domain_data['expiration_date'])
-        if 'updated_date' in domain_data:
-            domain_data['updated_date'] = parse_domain_date(domain_data['updated_date'])
-
-        dbot_score = Common.DBotScore(
-            indicator=indicator_domain,
-            integration_name='Pulsedive',
-            indicator_type=DBotScoreType.DOMAIN,
-            score=score,
-            malicious_description=f'Pulsedive returned reputation {reputation}'
-        )
-
-        domain_standard_context = Common.Domain(
-            domain=indicator_domain,
-            # creation_date=domain_data.get('creation_date', None),
-            # expiration_date=domain_data.get('expiration_date', None),
-            # updated_date=domain_data.get('updated_date', None),
-            # organization=domain_data.get('org', None),
-            # name_servers=domain_data.get('name_servers', None),
-            # registrant_name=domain_data.get('name', None),
-            # registrant_country=domain_data.get('country', None),
-            # registrar_name=domain_data.get('registrar', None),
-            dbot_score=dbot_score
-        )
-
-        command_results.append(CommandResults(
-            readable_output=tableToMarkdown('Domain Details:', domain_data),
-            outputs_prefix='Pulsedive.Domain',
-            outputs_key_field='indicator',
-            outputs=domain_data,
-            indicator=domain_standard_context
-        ))
+        try:
+            domain_data = client.get_domain_reputation(domain, api_key)
+            indicator_domain = domain_data['indicator']
+            reputation = domain_data['risk']
+            score = convert_to_xsoar_severity(reputation)
+
+            if 'creation_date' in domain_data:
+                domain_data['creation_date'] = parse_domain_date(domain_data['creation_date'])
+            if 'expiration_date' in domain_data:
+                domain_data['expiration_date'] = parse_domain_date(domain_data['expiration_date'])
+            if 'updated_date' in domain_data:
+                domain_data['updated_date'] = parse_domain_date(domain_data['updated_date'])
+
+            dbot_score = Common.DBotScore(
+                indicator=indicator_domain,
+                integration_name='Pulsedive',
+                indicator_type=DBotScoreType.DOMAIN,
+                score=score,
+                malicious_description='Pulsedive returned reputation {reputation}'
+            )
+
+            domain_standard_context = Common.Domain(
+                domain=indicator_domain,
+                # creation_date=domain_data.get('creation_date', None),
```

Comment:
Is this commented out code left here by intention ?

Reply:
yes,
the parsing needs to be done in the feature.